src/internal/operations/backup.go

```diff
@@ -502,7 +526,7 @@ func consumeBackupCollections(
 	if kopiaStats.ErrorCount > 0 {
 		err = clues.New("building kopia snapshot").WithClues(ctx)
 	} else if kopiaStats.IgnoredErrorCount > kopiaStats.ExpectedIgnoredErrorCount {
-		err = clues.New("downloading items for persistence").WithClues(ctx)
+		logger.Ctx(ctx).Info("recoverable errors were seen during backup")
```

Comment:
Note: I've removed primary failure from here. Thought process behind this:
1. We set[ kopia error handling policy](https://github.com/alcionai/corso/blob/34d6c18d74698b3ec8e0318e855dd498d4e93bf7/src/internal/kopia/wrapper.go#L284) to ignore file & directory errors.
2. Ignored errors are recoverable, [as marked here](https://github.com/alcionai/corso/blob/34d6c18d74698b3ec8e0318e855dd498d4e93bf7/src/internal/kopia/upload.go#L290) in corso.
3. Recoverable errors should not be counted as a primary failure. 
4. Presence of recoverable errors (and absence of primary failure) is a key requirement for persisting assist backups.